x11-misc/qt5ct/files/qt5ct-1.3-cmake-dbus.patch

```diff
@@ -0,0 +1,36 @@
+diff -ur a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Has this been submitted upstream?

Reply:
I haven't submitted it myself.  Not sure about the patch's author (Nuna Silva), although I don't seen any obvious sign of it under the upstream home page.

I don't know cmake at all.  Is there a cleaner way to code for optional dependencies, perhaps involving some kind of additional variable and/or something more like a C '?:' operator on a single line?